tests/spread/tools/restore.sh

```diff
@@ -7,7 +7,7 @@ apt-get autoremove --purge -y
 snaps="$(snap list | awk '{if (NR!=1) {print $1}}')"
 for snap in $snaps; do
 	case "$snap" in
-		"core" | "core16" | "core18" | "snapcraft")
+		"core" | "core16" | "core18" | "snapcraft" | "multipass")
```

Comment:
there might be a chance for races here, so let's keep an eye on this (recording tests with exact matching assertions).